src/index.ts

```diff
@@ -169,8 +172,8 @@ async function resolveTarStreamOnFinish(
 			'error',
 			new Error(
 				dockerfile
-					? `Specified dockerfile could not be resolved: ${dockerfile}`
-					: 'Resolution could not be performed',
+					? `Specified file not found or is invalid: ${dockerfile}`
+					: 'Could not find a Dockerfile for this project',
```

Comment:
A project = app can have several services and one Dockerfile per service, right? If the Dockerfile is missing for one of the services, I think this message would printed. So perhaps replace the word "project" with "service"?

Reply:
Good point